nanoFramework.System.Net.Http/Http/System.Net.HttpWebRequest.cs

```diff
@@ -1948,7 +1948,8 @@ private byte[] GetHTTPRequestData()
             }
             else
             {
-                statusLine = Method + " " + Address.AbsolutePath + " HTTP/" + ProtocolVersion + "\r\n";    // .PathAndQuery
+                // statusLine = Method + " " + Address.AbsolutePath + " HTTP/" + ProtocolVersion + "\r\n";    // .PathAndQuery
+                statusLine = Method + " " + Address.PathAndQuery + " HTTP/" + ProtocolVersion + "\r\n";
```

Comment:
```suggestion
                statusLine = Method + " " + Address.PathAndQuery + " HTTP/" + ProtocolVersion + "\r\n";  // .PathAndQuery
```

Reply:
No need to keep the wrong code commented. Better remove it.